node/tm-helpers/package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "tm-helpers",
```

Comment:
```suggestion
    "name": "@thingweb/tm-helpers",
```
but I feel we should find a nicer name. Any opinions @relu91 ?

Reply:
why don't we just call it `@thingweb/thing-model` ? It's clean but it might be mistaken as if we are offering a thing-model of @thingweb. I think we can solve this mistake with a simple description and readme.

Reply:
Now using this name :) The readme explains that it is helper tools to do some functions